docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,865 @@
+================= ================== ================== ==================
+
+
+Table 4. Tube-Length correction factor.
+
+===================    =====
+HX Tube Length (ft)    FL
+===================    =====
+8                      1.25
+12                     1.12
+16                     1.05
+20                     1.00
+===================    =====
+
+Construction material correction factor (FM_Mat) can be computed with Eq. 22.44 (Seider et al.)
+
+.. math:: material\_factor = a + (\frac{area}{100})^{b}
+
+
+Table 5. Materials of construction factors.
+
+================================== ====== ======
+Materials of Construction
+Shell / Tube                       a      b
+================================== ====== ======
+carbon steel/carbon steel          0.00   0.00
+carbon steel/brass                 1.08   0.05
+carbon steel/stainless steel       1.75   0.13
+carbon steel/monel                 2.1    0.13
+carbon steel/titanium              5.2    0.16
+carbon steel/Cr-Mo steel           1.55   0.05
+Cr-Mo steel/Cr-Mo steel            1.7    0.07
+stainless steel/stainless steel    2.7    0.07
+monel/monel                        3.3    0.08
+titanium/titanium                  9.6    0.06
+================================== ====== ======
+
+Pressure Changer Cost
+^^^^^^^^^^^^^^^^^^^^^
+
+The costing of a pressure changer unit model is more complicated, because the pressure changer 
+model can be imported into the flowsheet object representing a pump, turbine, compressor, or a 
+simply pressure changer (fan, blower, etc.). The `cost_pressure_changer` method currently supports costing of pumps, turbines, and compressors. The method authomatically interrogates the flowsheet object to determine if the unit is being used as a pump, turbine, or compressor. 
+
+The `cost_pressure_changer()` method authomatically determines if the unit model is being used as a pump, 
+turbine, or compressor based on the `compressor` and `thermodynamic_assumption` configuration 
+arguments provided by the user where creating the unit model. A summary of the decision logic is shown below.
+
+
+========== =========== =================================
+Unit Type  compressor  thermodynamic_assumption
+========== =========== =================================
+Turbine    False       Any
+Pump       True        pump
+Mover      True        not pump, isothermal or adiabatic
+========== =========== =================================
+
+Additionally, some unit types have different sub-types which can be costed appropiately. In these cases, 
+an additional argument is provided to `cost_pressure_changer` to identify the sub-type to use which is detailed below.
+
+Turbine Cost Model
+""""""""""""""""""
+The turbine cost is based on the mechanical work of unit (work_mechanical), this correlation has been obtained using the NETL Report (DOE/NETL 2015).
+
+.. math:: self.costing.purchase\_cost = 580*(mechanical_{work})^{0.81}
```

Comment:
Check the numbers here - the code appears to use `530` instead of `580`.

Further, seeing as this comes from a different source, I must ask if it really belongs here (I was not aware of this when converting the costing package). We call this the SSLW example, so it seems odd to include methods from outside this.